.pyproject_generation/pyproject_custom.toml

```
@@ -1,11 +1,11 @@
 [project]
 name = "dlqs"
-version = "2.0.0"
+version = "3.0.0"
```

```suggestion
version = "2.0.0"
```
We haven't released version 2.0.0 yet, so we don't need to change the number.